bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SyncThread.java

```diff
@@ -60,6 +62,7 @@
 @Slf4j
 class SyncThread implements Checkpointer {
 
+    @Getter(AccessLevel.PACKAGE)
```

Comment:
why even add a getter? it's final

Reply:
It is just personal practice. I generally dislike referencing field directly, which is not friend to testing at all.

Reply:
Sure. Not a fan of lombok, which is why I asked. We should eventually make construction of all these executors at the top level and pass them in via the constructor so we can mock them and get rid of non-determinism.